experimental/waveshare_eink/linux_kernel_modules/fb_waveshare_eink.c

```diff
@@ -338,6 +335,8 @@ static int ws_eink_init_display(struct ws_eink_fb_par *par)
 
 static int ws_eink_update_display(struct ws_eink_fb_par *par)
 {
+	ws_eink_reset(par);
+
 	int ret = 0;
```

Comment:
variable declarations must be at the beginning of a scope due to the compiler settings that are used when building this module.